Netezos.Tests/Forging/LocalForgingTest.cs

```diff
@@ -16,7 +17,7 @@ namespace Netezos.Tests.Forging
     public class LocalForgingTest
```

Comment:
I am wondering if there's interest for unit tests for each of the unforge operations (int, string, array, etc.). But those methods are not public in `LocalForge` so cannot be accessed here. How would we like to handle that?

Reply:
I don't think we need that right now, because all of those is already covered by `ForgeOperationGroupAsync`.